.github/actions/all-in-one/action.yml

```diff
@@ -0,0 +1,9 @@
+name: 'create all-in-one'
```

Comment:
[nit] can we rename `all in one` everywhere (directory name, job names) to something more meaningful? Maybe we can refer to the fact that it generates installation scripts? "generate_installation_scripts" or whatever

Reply:
gen-install-scripts